package.json

```
@@ -213,7 +213,7 @@
     "@cowprotocol/contracts": "^1.3.1",
     "@cowprotocol/cow-js": "^1.0.0-RC.0",
     "@cowprotocol/cow-runner-game": "^0.2.9",
-    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#0.0.0-rc4-artifacts",
+    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#1.0.0-artifacts",
```

This changed in the latest release. :see_no_evil: 

```suggestion
    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#v1.0.0-artifacts",
```

---

Hmm, how did this work then? o.O

---

@fedgiac , why the tag is not just `v1.0.0`?

---

> why the tag is not just v1.0.0?

Every time some code is tagged with tag `t`, then CI generates a new commit that creates all content that doesn't belong to the repo code itself but it's still useful in a release. This is tagged as `t-artifacts` (the name is because we mostly generate the contract build artifacts). Since here we need code that is not part of the repo then `artifacts` is needed.